kde-apps/libkleo/libkleo_kf6-24.12.3.recipe

```diff
@@ -51,13 +51,25 @@ REQUIRES="
 	lib:libQt6Widgets$secondaryArchSuffix
 	lib:libQt6Xml$secondaryArchSuffix
 	"
+REPLACES="
+	libkleo24$secondaryArchSuffix
+	"
 
 PROVIDES_devel="
-	libkleo24${secondaryArchSuffix}_devel = $portVersion
+	libkleo_kf6${secondaryArchSuffix}_devel = $portVersion
 	devel:libKPim6Libkleo$secondaryArchSuffix = $libVersionCompat
 	"
 REQUIRES_devel="
-	libkleo24$secondaryArchSuffix == $portVersion base
+	libkleo_kf6$secondaryArchSuffix == $portVersion base
+	"
+
+ARCHITECTURES_doc="any"
```

Comment:
Just a heads up: I have planned some changes for this pattern in the future, if https://github.com/haikuports/haikuporter/pull/321 is getting merged like that. See #12054 as an example how it should look like. (The `REQUIRES_doc` entry will probably have to be removed entirely in this case.)

I saw quite a few KDE packages that are affected by this.